drupal/init-env.php

```diff
@@ -11,6 +11,40 @@
  */
 
 use Drupal\consumers\Entity\Consumer;
+use Drupal\pathauto\Entity\PathautoPattern;
+use Drupal\pathauto\PathautoState;
+use Drupal\taxonomy\Entity\Term;
+use Drupal\taxonomy\Entity\Vocabulary;
+
+// Ensure every taxonomy vocabulary has a Pathauto pattern, so that tests can
+// have reliable URLs to visit.
+$vocabularies = Vocabulary::loadMultiple();
+// Article types already have a pattern.
+unset($vocabularies['article_type']);
+/** @var \Drupal\taxonomy\VocabularyInterface[] $vocabulary */
+foreach ($vocabularies as $id => $vocabulary) {
+  /** @var \Drupal\pathauto\PathautoPatternInterface $pattern */
+  $pattern = PathautoPattern::create([
+    'id' => $id,
+    'label' => $vocabulary->label(),
+    'type' => 'canonical_entities:taxonomy_term',
+  ]);
+  $pattern->addSelectionCondition([
+    'id' => 'entity_bundle:taxonomy_term',
+    'context_mapping' => [
+      'taxonomy_term' => 'taxonomy_term',
+    ],
+    'bundles' => [
+      $id => $id,
+    ],
+  ]);
+  $pattern->setPattern("/$id/[term:name]")->save();
```

Comment:
Since these patterns are just used to keep our tests stable, it doesn't really matter if they're not super realistic.